src/legal-officer/transaction-protection/LocDetails.tsx

```diff
@@ -0,0 +1,18 @@
+import { LocContextProvider } from "./LocContext";
+import ContextualizedLocDetails from "./ContextualizedLocDetails";
+import { UUID } from "../../logion-chain/UUID";
+import { useParams } from "react-router";
+
+export interface Props {
+    backPath: string,
+}
+
+export default function LocDetails(props: Props) {
+    const locId: UUID = new UUID(useParams<{ locId: string }>().locId);
+
+    return (
+        <LocContextProvider locId={ locId }>
```

Comment:
I think you can move the provider at the same level as the `UserContextProvider` and thus have one single `LocDetails` component without the need to "contextualize" it with an additional component.

Reply:
Sorry, I missed the fact that the provider was linked to a single LOC at a time.